src/integrationTest/java/org/opensearch/security/api/InternalUsersRestApiIntegrationTest.java

```diff
@@ -149,13 +151,17 @@ static ToXContentObject internalUser(
                 builder.field("attributes", attributes);
             }
             if (securityRoles != null) {
-                builder.field("opendistro_security_roles");
+                builder.field(getRoleField());
                 securityRoles.toXContent(builder, params);
             }
             return builder.endObject();
         };
     }
 
+    private static String getRoleField() {
+        return randomFrom(List.of(OPENDISTRO_SECURITY_ROLES, DIRECT_SECURITY_ROLES));
```

Comment:
I see that there are also other tests which use `randomFrom`. However, I am kind of skeptical of using random sources in tests, especially if the random space is only two elements. This has high potential to lead to test flakyness.

Shoudlnt we just test both cases?

Reply:
I am okay with testing both cases for this change.